extra/3-magic-8-ball.js

```diff
@@ -59,9 +88,25 @@ function shakeBall() {
   This function should expect to be called with any value which was returned by the shakeBall function.
 */
 function checkAnswer(answer) {
-  //Write your code in here
+  let message = "";
+  for (let i = 0; i < choicesOfAnswer.length; i++) {
+    if (answer === choicesOfAnswer[i]) {
+      message = i;
+    }
+  }
+  if (message < 5) {
+    return "very positive";
+  } else if (message >= 5 && message < 10) {
+    return "positive";
+  } else if (message >= 10 && message < 15) {
```

Comment:
For lines 99 and 101 - do you need 2 parts to your if statement? Have you already established one of them?